egs2/mucs21_subtask1/asr1/conf/specaug.yaml

```diff
@@ -0,0 +1,16 @@
+process:
```

Comment:
you don't need this file.

Reply:
in espnet2, we usually put specaug related options in the training config.
Please check it and test the effectiveness of specaug.